### DISCUSSION TITLE
Немає жодного контакту з іншими людьми

### DISCUSSION THREAD

**Discussion by demycat** [Ludeon/RimWorld-Ukrainian#33] [General]
Доброї ночі, мене звати Владислав і я хотів дізнатися с приводу допомоги в перекладі.
В мене виникли проблеми в розумінні, як працює ця схема, бо якогось єдиного голосового чату у вас немає із за чого я просто загублений і не знаю, що куди і як.
Як я зрозумів, переклад краще робити на crowdin, бо там доволі прості інструменти для цієї роботи, але мене здивувало, що активність там була за останній місяць і коли спробував розібратись у файлах об'єктів, які не були перекладені у грі, то на сайті є переклад, але чомусь його немає у самій грі.
Це вводить мене в ступір, бо я спершу думав, що перекладом взагалі не займаються, але на ділі він є, но чомусь її немає офіційно у грі
Можно з кимось зв'язатись по голосовій(Discord наприклад), щоб мені могли пояснити, що і до чого...
Буду вдячний за любу допомогу)

**Comment by dmytrokyrychuk** [comment]
Дзвоніть на Dmytro#0488, я поясню, що знаю.

**Comment by MartianDill** [comment]
є чат в Телеграмі
https://t.me/+Tposfrg1AwU5N2I6
